queries/objdump/highlights.scm

```diff
@@ -0,0 +1,20 @@
+(address) @number
+(byte) @constant
+(hexadecimal) @number
+(integer) @number
+(section_address) @number
+
+(disassembly_section_label) @text.title
+[(file_offset) (discriminator)] @field
+(file_path) @string
+(identifier) @namespace
```

Comment:
```suggestion
(identifier) @variable
```
Maybe?

Reply:
Hm. You could make a case for either I think

`(identifier)` comes up during section labels

https://github.com/ColinKennedy/tree-sitter-objdump/blob/master/test/corpus/sections.txt#L5-L8
```
Disassembly of section .text:

0000000000000000 <engine::world::World::createEntity()> (File Offset: 0x70):
   0:	41 55                                                 	push   r13
  1f0:	48 83 c3 08                                           	add    rbx,0x8
```

Everything under `.text` is part of its contents. And the lines indented `engine::world::World::createEntity()` are byte offsets from the starting address of `0000000000000000`. It felt very group-y to me. I use `(identifier)` for both `.text`  and `engine::world::World::createEntity()` so that's what led to `@namespace`. Maybe you'd prefer if I change the query to reflect the nesting? e.g. something like

```scm
(disassembly_section_label
  (identifier) @namespace)
(identifier) @variable
```

So if it's grouped, it's a namespace. But if it's free-form, it's a different highlight. What do you think?

Reply:
That sounds better than the current queries

Reply:
The free-form one should be on top.

Reply:
> (identifier) @variable

> The free-form one should be on top.


This is done